src/CMakeLists.txt

```diff
@@ -3,13 +3,23 @@ project(ki017)
 
 set(SOURCE
         main.cpp
+		AI.cpp
+		AICallback.cpp
+		util/Logging.cpp
+		generate/ItemChoice.cpp
+		generate/EquipmentChoice.cpp
+		generate/GameOperation.cpp
+		generate/random/ItemChoice.cpp
+		generate/random/EquipmentChoice.cpp
+		generate/random/GameOperation.cpp
```

Comment:
Einrückung bitte fixen (sind das 2 tabs statt 8 spaces? in github nicht besonders gut erkennbar)